src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -128,10 +136,23 @@ private void setupListeners() {
     port.getDocument().addDocumentListener(listener);
     password.getDocument().addDocumentListener(listener);
     toAddress.getDocument().addDocumentListener(listener);
+    passwordSaved.addActionListener(e -> passwordSavedChanged());
     useTls.addActionListener(e -> fireEditorChanged());
     testEmail.addActionListener(e -> testEmail());
   }
 
+  private void passwordSavedChanged() {
+    fireEditorChanged();
+
+    if (passwordSaved.isSelected()) {
+      GameRunner.showMessageDialog(
+          "Your password will be stored unencrypted in the local file system. "
+              + "You should not choose to remember your password if this makes you uncomfortable.",
+          GameRunner.Title.of("Security Warning"),
```

Comment:
Really scary text to make the user nervous. :smile:  Please suggest improvements.  (Identical text appears in `ForumPosterEditor`.)